blog/content/post/2025-07-xx-edk2-on-odroid-m2.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Booting EDK2 on Odroid M2
+abstract: ''
+cover: /img/tianocore_logo.jpg
+author: michal.kopec
+layout: post
+published: true
+date: 2025-07-01
+archives: "2025"
+
+tags:
+  - QubesOS
+categories:
+  - Firmware
+
+---
+
+This post describes the process of booting TianoCore EDK II on an Odroid M2
+platform (ARM64, RK3588 based) and gives an overview of the status of ARM
+platforms in EDK II.
+
+## Introduction
+
+TianoCore EDK II is the reference implementation of UEFI and is the de facto
+standard boot firmware in the x86 space. In the ARM world, especially in the
+embedded Linux space, that role is typically served by U-boot. With the rise of
+high performance ARM computing, however, EDK II is becoming an increasingly
+popular option for system builders. In this blog post I'll be checking out the
+process of booting EDK II and an UEFI OS on an Arm platform with pre-existing
```

Comment:
I was wondering if this is a conscious distinction between ARM and Arm?

Reply:
I don't know how correct it is, but my approach always was that ARM is a computer architecture (as spelled in most FOSS resources) and Arm is a company name: https://en.wikipedia.org/wiki/ARM_architecture_family